### DISCUSSION TITLE
Why change from semver?

### DISCUSSION THREAD

**Discussion by omnigenous** [vmiklos/plees-tracker#445] [Q&A] [answered]
Why change from Semantic Versioning to this 👉 `2024.02.0`? At first I thought that app was compromised when I saw this.

**Comment by vmiklos** [comment, marked as answer]
In fact I already cut releases based on calendar:

- 7.6 was tagged in Aug 2023
- 7.5 was tagged in Feb 2023
- 7.4 was tagged in Aug 2022
- 7.3 was tagged in Feb 2023

so this is just adjusting the versioning to match reality. :-)

But you're right, it is worth to note this in the release notes, see https://github.com/vmiklos/plees-tracker/pull/447
